src/recaplog.8

```diff
@@ -17,32 +17,32 @@
 .\" You should have received a copy of the GNU General Public
 .\" License along with this manual; if not, write to the Free
 .\" Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
-.\" Boston, MA 02110-1301 USA.
+.\" Boston, MA 02110\-1301 USA.
```

Comment:
I don't think this is needed, the license is never shown in the man page.

Reply:
Fixed, thanks!